### PR TITLE
Frontend Payment Integration

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- Added stripe payment integration
- Added payment success page
- Coupon is integrated with stripe, 2 coupon can be added while checking out i.e. in promocode section only when the price is greater than 20000 and 40000

- PS : It will not work until I merge the temp branch in dev, attaching gif, success page will also be visible once it is merged with dev branch
- You can check the success page with this url - https://weebo-frontend-git-feat-payment-integration-sukanyasen530.vercel.app/checkout/success

Payment

### CODE REVIEW

**Review by biswanath-tewari — APPROVED** [review]

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -53,13 +53,17 @@ const Navbar = () => {
...
+        {wishlistItems?.length > 0 && (
+          <span className="badge__count"> {wishlistItems?.length} </span>
+        )}
       </Link>
 
       <Link to="/cart" className="btn-icon badge">
         <AiOutlineShoppingCart className="btn-icon__icon" />
         <span className="btn-icon__text">Cart</span>
-        <span className="badge__count"> {cartItems.length} </span>
+        {cartItems?.length > 0 && (
```
Comment (Mohitkumar33) [inline src/components/Navbar/Navbar.jsx]: ? : instead  of && may it may be giving error on console

File: src/services/products.js [hunk 2]
```diff
@@ -15,7 +15,9 @@ export const loadProducts = async (dispatch) => {
         payload: data?.products || [],
       });
     }
+
   } catch (e) {
+    console.log(e);
```
Comment (Mohitkumar33) [inline src/services/products.js]: console.error instead of console.log

**Review by Mohitkumar33 — APPROVED** [review]
OMG. Code is very clean with lot of features. How can you write this much clean code. Good job

File: src/services/products.js (hunk 2 above)
Comment (SukanyaSen530) [reply src/services/products.js]: Removed the log - [e4f6c13](https://github.com/SukanyaSen530/WeebO-frontend/pull/11/commits/e4f6c1359a136c3468a711cafd7c72212e8396e0)

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (SukanyaSen530) [reply src/components/Navbar/Navbar.jsx]: fixed - [e4f6c13](https://github.com/SukanyaSen530/WeebO-frontend/pull/11/commits/e4f6c1359a136c3468a711cafd7c72212e8396e0)

**Pull request merged**